libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateAlliedProjectile.hpp

```diff
@@ -22,6 +22,7 @@
 #include "R-TypeLogic/Global/Components/SizeComponent.hpp"
 #include "R-TypeLogic/Global/Components/VelocityComponent.hpp"
 #include "R-TypeLogic/Global/Components/WeightComponent.hpp"
+#include "R-TypeLogic/Global/Components/PlayerComponent.hpp"
```

Comment:
Why ?

Reply:
entity.addComponent<AlliedProjectile>(ally.getComponent<Player>().playerIdentifier); in cpp l33